api/events/models.py

```diff
@@ -31,54 +31,40 @@ class Events(Base, Utility):
     app_booking = Column(Boolean, nullable=True, default=False)
 
 
-def filter_events_by_date_range(query, start_date, end_date):
+def filter_event(start_date, end_date, room_id=None):
     """
-    Returns events that fall within the date range supplied
+    Returns filtered events if the room id
+    or start date and end date are supplied,
+    and return all events when those parameters are not supplied.
```

Comment:
It looks good, please reword the last line of this docstring. It suggests functionality contrary to what the function does.